packages/@n8n/db/src/entities/execution-data.ts

```diff
@@ -21,8 +21,9 @@ export class ExecutionData {
 	@PrimaryColumn({ transformer: idStringifier })
 	executionId: string;
 
-	@ManyToOne('ExecutionEntity', 'data', {
-		onDelete: 'CASCADE',
+	@OneToOne('ExecutionEntity', 'executionData')
+	@JoinColumn({
+		name: 'executionId',
```

Comment:
Why are we removing `ON DELETE CASCADE`? Won't this open the door for orphan execution data rows?

Reply:
I think this setting has no influence on the actual behavior, because the foreign keys in the DB do have the on delete cascade rule.